modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/layer/RasterMBLayer.java

```diff
@@ -225,10 +225,17 @@ public Number getFadeDuration() {
 
         // Use of builder is easier for code examples; but fills in SLD defaults
         // Currently only applies the opacity.
+        if(getBrightnessMax() != null || getBrightnessMin() != null){
```

Comment:
Ah, I do think that you are right after testing vs. the intended results on mapbox studio.  In this respect I believe that this would be better used for the contrast property.  That being said, I am still having trouble finding a good example where a whole image is transformed. It looks like currently it can only work band by band - resulting in grayscale images.